packages/amplify-e2e-core/src/index.ts

```diff
@@ -91,9 +94,19 @@ export async function installAmplifyCLI(version = 'latest') {
     env: process.env,
     stdio: 'inherit',
   });
-  process.env.AMPLIFY_PATH = process.platform === 'win32'
+  
+  console.log("SETTING PATH:");
+  if(version === '10.5.1'){
+    process.env.AMPLIFY_PATH = process.platform === 'win32'
+    ? path.join(os.homedir(), '.amplify', 'bin', 'amplify')
+    : path.join(os.homedir(), '.amplify', 'bin', 'amplify');
+  } else {
+    process.env.AMPLIFY_PATH = process.platform === 'win32'
     ? path.join(os.homedir(), '..', '..', 'Program` Files', 'nodejs', 'node_modules', '@aws-amplify', 'cli', 'bin', 'amplify')
     : path.join(os.homedir(), '.npm-global', 'bin', 'amplify');
+  }
```

Comment:
I had to update this because during setup, it would set AMPLIFY_PATH to the old cli location (before we moved to packaged cli), and this would ultimately fail because on v10 the cli is not installed to .npm-global